src/rdkafka.h

```diff
@@ -3726,6 +3742,19 @@ rd_kafka_incremental_unassign (rd_kafka_t *rk,
                                *partitions);
 
 
+/**
+ * @brief The rebalance protocol currently in use. This will be
+ *        RD_KAFKA_REBALANCE_PROTOCOL_NONE if the consumer has not
+ *        (yet) joined a group, else it will match the rebalance
+ *        protocol of the configured assignor(s), which must all
+ *        align. Switching between assignors with different
+ *        rebalance protocols without a restart is not currently
+ *        supported.
+ */
+rd_kafka_rebalance_protocol_t
+rd_kafka_rebalance_protocol (rd_kafka_t *rk);
```

Comment:
it's needed by the .net client, because this client's api automatically calls assign / incremental assign etc if the user doesn't. it needs to know which assign call variant to call.